tests/test_semdedup.py

```diff
@@ -658,7 +673,7 @@ def test_sematnic_cluster_level_dedup(self, tmpdir, which_to_keep, sim_metric):
 
         # Check content of semdedup_pruning_table with the filter matches the unique_ids
         semdedup_pruning_tables_df_filtered = semdedup_pruning_tables_df[
-            semdedup_pruning_tables_df["cosine_sim_score"] < 1 - 0.01
+            semdedup_pruning_tables_df["cosine_sim_score"] >= 1 - 0.01
```

Comment:
Can we do a quick test here for perform_removal=True, perform_removal=False here too.

Reply:
This is just testing one function which is `semantic_cluster_level_dedup`. The `test_semdedup` tests the perform_removal there.

Reply:
Cool. Thanks for clarifying.